### DISCUSSION TITLE
API for dev tools

### DISCUSSION THREAD

**Discussion by GlassGruber** [violentmonkey/violentmonkey#1656] [Q&A] [answered]
Hei there, is it possible to hook on API for dev tools listed for example here 
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/devtools
or here?
https://developer.chrome.com/docs/extensions/reference/devtools_network/

Are there any alternatives? The specific case I'm facing is trying to trigger a HAR export just like from the network tab, but I'm wondering in general if it's possible to play with dev tools API.

Thank you 🎩👌

**Comment by tophf** [comment, marked as answer]
Not possible. For personal use you can write a devtools extension yourself.

**Reply by GlassGruber** [reply]
I thought that might be the case, that's unfortunate, thank you !
